css/media.css

```
@@ -0,0 +1,110 @@
+@media (max-width:1080px){
+    .social-network{
```

```suggestion
    .social-network {
```
не забывай про пробелы после селектора, пройдись везде и поправь

---

Выполнил